dataplane/meson.build

```diff
@@ -1,7 +1,7 @@
 dependencies = []
 dependencies += libdpdk.get_variable('dpdk_dep')
 dependencies += libjson.get_variable('nlohmann_json_dep')
-dependencies += libchash.get_variable('balancer_dep')
+dependencies += libchash.get_variable('chash_dep')
 dependencies += dependency('libsystemd')
```

Comment:
Fixup that commit so that the commit introducing the build error no longer causes it